server/src/main/java/com/linecorp/centraldogma/server/internal/storage/repository/cache/CacheableQueryCall.java

```diff
@@ -36,14 +36,16 @@ final class CacheableQueryCall extends CacheableCall<Entry<?>> {
 
     final Revision revision;
     final Query<?> query;
+    final int includeLastFileRevision;
     final int hashCode;
 
-    CacheableQueryCall(Repository repo, Revision revision, Query<?> query) {
+    CacheableQueryCall(Repository repo, Revision revision, Query<?> query, int includeLastFileRevision) {
         super(repo);
         this.revision = requireNonNull(revision, "revision");
         this.query = requireNonNull(query, "query");
+        this.includeLastFileRevision = includeLastFileRevision <= 1 ? -1 : includeLastFileRevision;
```

Comment:
Do we not need to check equality based on `includeLastFileRevision` as well?

Reply:
That's my mistake.